src/gui/accountmanager.cpp

```diff
@@ -364,22 +362,20 @@ void AccountManager::displayMnemonic(const QString& mnemonic)
 
 void AccountManager::shutdown()
 {
-    auto accountsCopy = _accounts;
+    const auto accountsCopy = _accounts;
     _accounts.clear();
-    foreach (const auto &acc, accountsCopy) {
+    for (const auto &acc : accountsCopy) {
         emit accountRemoved(acc.data());
         emit removeAccountFolders(acc.data());
     }
 }
 
 bool AccountManager::isAccountIdAvailable(const QString &id) const
 {
-    foreach (const auto &acc, _accounts) {
-        if (acc->account()->id() == id) {
-            return false;
-        }
-    }
-    return true;
+    const auto it = std::find_if(_accounts.cbegin(), _accounts.cend(), [id](const auto &acc) {
```

Comment:
Could be `std::any_of`

Reply:
Yeah, or probably `std::none_of`... Then we don't have to negate the result from `std::any_of`. Seems semantically slightly better to me.